widgets/refinement-list.js

```diff
@@ -75,6 +75,13 @@ function refinementList({
   }
 
   return {
+    init: (state, helper) => {
+      // set the maxValuesPerFacet to max(limit, currentValue)
+      var maxValuesPerFacet = helper.getQueryParameter('maxValuesPerFacet');
```

Comment:
Settings of the initial helper configuration of the helper should go in getConfiguration.